src/cmds/docs/create.js

```diff
@@ -0,0 +1,164 @@
+const chalk = require('chalk');
+const fs = require('fs');
+const path = require('path');
+const config = require('config');
+const crypto = require('crypto');
+const frontMatter = require('gray-matter');
+const { promisify } = require('util');
+const { getProjectVersion } = require('../../lib/versionSelect');
+const fetch = require('../../lib/fetch');
+const { cleanHeaders, handleRes } = require('../../lib/fetch');
+const { debug } = require('../../lib/logger');
+
+const readFile = promisify(fs.readFile);
+
+module.exports = class CreateDocCommand {
+  constructor() {
+    this.command = 'docs:create';
```

Comment:
Can you rename this command to `docs:single`? Technically "create" is a bit of a misnomer since this command can create or update a doc.